google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -685,6 +692,10 @@ def clone(
                 Optional. The mapping from runtime parameter names to its values that
                 control the pipeline run. Defaults to be the same values as original
                 PipelineJob.
+            input_artifacts (Dict[str, str]):
+                Optional. The mapping from runtime input artifact names to its id that
+                control the pipeline run. Defaults to be the same values as original
+                PipelineJob.
```

Comment:
For example there is an existing model artifact on MLMD`"projects/123/locations/us-central1/metadataStores/default/artifacts/456"`, the mapping would look like user inputs `{"vertex_model": "456"}`, and pipeline proto is also expecting the artifact id without project and locations. Because the project id and locations must be the same as where pipeline job is running.

Reply:
Done.